tests/serde_roundtrips.rs

```diff
@@ -120,3 +120,40 @@ fn deserialize_too_many_fields() {
 
     assert!(song.is_ok(), "{:?}", song.unwrap_err());
 }
+
+#[test]
+fn deserialize_partial_level() {
+    let level = dash_rs::from_robtop_str::<Level<_, _>>(DARK_REALM_DATA);
+
+    assert!(level.is_ok(), "{:?}", level.unwrap_err());
+
+    let mut level = level.unwrap();
+
+    assert!(level.description.as_mut().unwrap().process().is_ok());
+}
+
+#[test]
+fn deserialize_level() {
+    let level = dash_rs::from_robtop_str::<Level<_, _>>(include_str!("data/11774780_dark_realm_gjdownload_response"));
+
+    assert!(level.is_ok(), "{:?}", level.unwrap_err());
```

Comment:
This is basically .unwrap, can be removed as it's done later too.

Reply:
Touche